src/main/java/com/ericsson/ei/controller/SubscriptionController.java

```diff
@@ -26,7 +26,7 @@
      * 
      */
     @RequestMapping(value = "", method = RequestMethod.GET)
-    public ResponseEntity<List<com.ericsson.ei.controller.model.Subscription>> getSubscriptions();
+    public ResponseEntity<List<String>> getSubscriptions();
```

Comment:
Why did you RAML generation removed Subscription? It should be either untyped list or use Subscription as type for the list